defcon/plugins/alertmanager.py

```diff
@@ -109,16 +135,23 @@ def render(template, data):
         env = jinja2.Environment()
         return env.from_string(template).render(**data)
 
-    def _to_status(self, root_alert, block, alert):
+    def _to_status(self, alert):
         """Return a status or None."""
         logging.debug('Handling %s' % (alert))
         if not self.match_labels(alert['labels'], self.labels):
             return None
         status = alert.get('status', {})
+
+        # Check Receiver
         if self.receiver is not None:
-            if 'routeOpts' in block:
-                if block['routeOpts']['receiver'] != self.receiver:
+            if alert.get('receivers') is not None:
```

Comment:
If no receivers are being assigned to an alert, it shouldn't be evaluated by defcon ( alert['receivers'] == None)